client/state/ui/guided-tours/selectors.js

```diff
@@ -74,6 +77,13 @@ const getTourFromQuery = createSelector(
 	getInitialQueryArguments
 );
 
+const DAY_IN_MILLISECONDS = 1000 * 3600 * 24;
```

Comment:
We _could_ use `moment`, which would allow us to get rid of this constant, and would read better.